feature-crowdloan-impl/src/main/java/jp/co/soramitsu/feature_crowdloan_impl/data/network/api/moonbeam/SignatureResponse.kt

```diff
@@ -0,0 +1,14 @@
+package jp.co.soramitsu.feature_crowdloan_impl.data.network.api.moonbeam
+
+import com.google.gson.annotations.SerializedName
+
+class SignatureResponse(
+    val address: String,
+    val contribution: String,
+    @SerializedName("previous-total-contribution")
```

Comment:
isn't there an option in gson to convert between cases by default? I believe there should be

Reply:
https://www.javadoc.io/doc/com.google.code.gson/gson/2.8.0/com/google/gson/FieldNamingPolicy.html

Reply:
Applicable to all moonbeam queries